test/libsolidity/SMTCheckerTest.cpp

```diff
@@ -118,6 +116,9 @@ SMTCheckerTest::SMTCheckerTest(std::string const& _filename):
 	else
 		BOOST_THROW_EXCEPTION(std::runtime_error("Invalid SMT invariant choice."));
 
+	if (m_modelCheckerSettings.invariants.invariants.empty())
+		m_expectations = removeInv(std::move(m_expectations));
+
```

Comment:
This needs to be AFTER we set the invariant setting in `m_modelCheckerSetting.invariants`.